assets/js/googlesitekit-idea-hub-notice.js

```diff
@@ -31,6 +31,7 @@ const editorNoticeKey = ( postID ) => {
 	return `modules::idea-hub::dismissed-editor-notice-${ postID }`;
 };
 
+// eslint-disable-next-line require-await
```

Comment:
Hmm, it doesn't look like this function needs to be `async`, as it doesn't return anything and there's no attempt to `catch` any exception it throws where it's invoked below...